framework/source/class/qx/ui/table/pane/Pane.js

```diff
@@ -283,7 +283,11 @@ qx.Class.define("qx.ui.table.pane.Pane",
       if (lastRow == -1 || lastRow >= paneFirstRow && firstRow < paneFirstRow + rowCount)
       {
         // The change intersects this pane
-        this.updateContent();
+        if (firstRow === lastRow) {
```

Comment:
Perhaps this should also test the number of rows in the model, if there is only one then we may as well do the full update.